File: URI 1011.cpp
#include <iostream>
#include <math.h>
#include <stdio.h>
using namespace std;

int main(){
    
    int r;
    
    std::cin >> r;
    
    printf("VOLUME = %.3f\n", (4.0/3.0)*3.14159*pow(r,3));
    return 0;
}

File: URI 1011.java
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		
		double a;
		Scanner input = new Scanner(System.in);
		a = input.nextDouble();
		
		System.out.printf("VOLUME = %.3f\n", (4 * 3.14159 * Math.pow(a, 3.0)) / 3);

	}
}